#single_led_preset_pattern

#Header header
uint16 led_number
uint8 led_preset_pattern 
